server/routes/urls.ts

```diff
@@ -0,0 +1,4 @@
+const BASE_SEARCH_URL = '/search'
+const SEARCH_PRISONER_URL = `${BASE_SEARCH_URL}/prisoner`
+
```

Comment:
There is a `config.ts` module in most services to get the environment variable values, or default the value for test running. I think it would be best to stick with the convention for providing these env values?